Makefile

```diff
@@ -87,7 +87,6 @@ run: fmt update-bazel
 	KUBE_PATCH_CONVERSION_DETECTOR=true \
 	KUBE_CACHE_MUTATION_DETECTOR=true \
 	bazel run //cmd/smith:smith_race \
-		--direct_run \
```

Comment:
`direct_run` is the default behavior starting from `0.14.0` and is deprecated in `0.15.0`